root/static/scripts/account/components/ApplicationForm.js

```diff
@@ -101,14 +101,21 @@ class ApplicationForm extends React.Component<Props, State> {
           options={oauthTypeOptions}
           required
         />
-        {this.state.form.field.oauth_type.value === 'web' ? (
-          <FormRowURLLong
-            field={this.state.form.field.oauth_redirect_uri}
-            label={addColon(l('Callback URL'))}
-            onChange={this.handleOauthRedirectURIChange}
-            required
-          />
-        ) : null}
+        <FormRowURLLong
```

Comment:
This has type="url" so it always auto-validates, meaning the example ("org.example.app") is not accepted.

Reply:
It was actually invalidated by `MusicBrainz::Server::Form::Field::OAuthRedirectURI`, now fixed with https://github.com/metabrainz/musicbrainz-server/pull/916/commits/3f84f7718c1f2c322b1fc2bb4b77bd0f871c164f#diff-e6382dac47170663e5de065f3af86848

Reply:
If I enter the example ("org.example.app") I still get "Please enter a URL."

It does work if I enter "org.example.app://" (with the "://" end) but we really shouldn't have a situation where the example we give is actually not allowed for entry.

Reply:
Oops, I meant to leave the comment at https://github.com/metabrainz/musicbrainz-server/pull/916#discussion_r259857876 for both strings. Can we just copy that here?

Reply:
I replaced the misleading URL sheme example in the input note with the full URL example I already fixed in the error message.